README.md

```diff
@@ -1,18 +1,62 @@
-Jenkins on Mesos
-================
-
-Jenkins standalone instance package with the Mesos plugin pre-installed
-
-
-# Universe Package
-
-Jenkins is available as a package in the [Mesosphere Multiverse](https://github.com/mesosphere/multiverse/). To make changes to the Jenkins package, submit a pull-request against the Multiverse.
+# Jenkins on Mesos
+Run a Jenkins master on Mesos and Marathon, using Docker and Tomcat.
+
+## Overview
+This repo contains a [Dockerfile](Dockerfile) that runs Jenkins inside the
+[Apache Tomcat][tomcat-home] Java application server. It also provides several
+Jenkins plugins and a basic Jenkins configuration, to get you up and running
+quickly with Jenkins on the DCOS.
+
+## Included in this repo
+Base packages:
+  * [Apache Tomcat][tomcat-home] 8.0.27
+  * [Jenkins][jenkins-home] 1.609.3 (LTS)
```

Comment:
It looks like that was released about a week ago. Let me try it out first, but I suspect that yes, we would want to ship the latest LTS here.